vector/src/main/java/im/vector/app/core/extensions/Session.kt

```diff
@@ -34,9 +34,21 @@ fun Session.configureAndStart(context: Context, startSyncing: Boolean = true) {
         startSyncing(context)
     }
     refreshPushers()
+    configureContentScanner()
     context.singletonEntryPoint().webRtcCallManager().checkForProtocolsSupportIfNeeded()
 }
 
+private fun Session.configureContentScanner() {
+    sessionParams.homeServerConnectionConfig.antiVirusServerUri
+            ?.let { url ->
+                contentScannerService().let {
```

Comment:
Also I would prefere to use a `run` or `with` as we don't care about the resulting value. Also you can store the service in a dedicated variable